tests/test_rotary_embeddings.py

```diff
@@ -21,6 +25,32 @@
 EMB = 32
 
 
+def test_helper_methods():
+    # rotate_half
+    tens = torch.tensor([[0, 1, 2, 3], [3, 1, 2, 0], [0, 1, 0, 1], [1, 0, 1, 0]])
```

Comment:
nit: may be nice to format this for readability like [here](https://github.com/facebookresearch/xformers/blob/37cf11f9c8124d439b89d7bb133b0157f11ecad3/tests/test_attention_patterns.py#L232)

Reply:
The pre-commit formatting configurations (black specifically) keep reverting my formatting changes back to what is shown above. @blefaudeux should we leave it as is?